codesign/bin/codesign.dart

```diff
@@ -34,13 +32,21 @@ const String kGcsUploadPathOption = 'gcs-upload-path';
 /// For [kGcsDownloadPathOption] and [kGcsUploadPathOption], they are required parameter to specify the google cloud bucket paths.
 /// [kGcsDownloadPathOption] is the google cloud bucket prefix to download the remote artifacts,
 /// [kGcsUploadPathOption] is the cloud bucket prefix to upload codesigned artifact to.
-/// For example, supply '--gcs-download-path=gs://flutter_infra_release/ios-usb-dependencies/unsigned/libimobiledevice/<commit>/libimobiledevice.zip',
+/// For example, supply '--GCS-DOWNLOAD-PATH=gs://flutter_infra_release/ios-usb-dependencies/unsigned/libimobiledevice/<commit>/libimobiledevice.zip',
```

Comment:
flags are preferred to be lowercase we are capitalizing env variable names only.

Reply:
Sure, I lowered case for file paths, and kept upper case for cert name and user name.